nodejs/script/build.ts

```diff
@@ -40,7 +40,7 @@ const command = CMAKE_JS_FULL_PATH;
 const args = [
   (REBUILD ? 'reconfigure' : 'configure'),
   '--arch=x64',
-  '--CDnapi_build_version=3',
+  '--CDnapi_build_version=6',
```

Comment:
Thank you.

Reply:
You're welcome :)

Reply:
The build issue is figured out in #4148 . by upgrading node_addon_api to v3.0.0 we are able to compile in all Node.js version using NAPI v3.